app/pages/sections/catalogue-solutions/order-item/controller.js

```diff
@@ -1,12 +1,33 @@
 /* eslint-disable no-restricted-globals */
-import { getData } from 'buying-catalogue-library';
+import { getData, postData } from 'buying-catalogue-library';
 import { getContext, getErrorContext } from './contextCreator';
 import { logger } from '../../../../logger';
 import { getEndpoint } from '../../../../endpoints';
 import commonManifest from './commonManifest.json';
 import { getSelectedPriceManifest } from './manifestProvider';
 import { getDateErrors } from '../../../../helpers/getDateErrors';
 
+const extractDeliveryDate = (detail) => {
```

Comment:
So we do something similar in the commencementDate section where we pass in the date from FE and convert it to a date we can pass to the API. How about creating a generic function and moving this into the helpers folder so it can be used by both?

Reply:
done